sunpy/tests/setup_command.py

```diff
@@ -117,6 +122,7 @@ def generate_testing_command(self):
                        online=online,
                        offline=offline,
                        figure=self.figure,
```

Comment:
I wonder if all of these should be self.*

Reply:
In the line 104 above, we did this
`online = self.online`

Reply:
So we could do the same for figure?

Reply:
I don't see any problem if we do same for figure and figure_only. Should I ?

Reply:
Sure.